utils/exceptions.py

```diff
@@ -15,28 +15,31 @@
 """
 
 class VFATDACBiasCannotBeReached(ValueError):
-    """
-    This exception is raised if DAC analysis determines that to achieve the 
-    nominal bias current/voltages in ``nominalDacValues`` dictionary, 
-    of: ``gempython.gemplotting.utils.anaInfo``, the DAC would need to be set
-    to a value outside of the DAC range ``[0, max]`` where ``max`` is
-    ``maxVfat3DACSize[dacSelect][0]``.  Info can be found on these ranges in
-    the ``maxVfat3DACSize`` dictionary of: ``gempython.tools.hw_constants``
-    """
+
     def __init__(self, message, errors):
         super(VFATDACBiasCannotBeReached, self).__init__(message)
 
         self.errors = errors
         return
 
-class VFATDACFitLargeChisquare(ValueError):
+class DACAnalysisException(ValueError):
     """
     This exception is raised when a DAC vs ADC fit returns a large chisquare value
 
     """
-    def __init__(self, message, errors):
-        super(VFATDACFitLargeChisquare, self).__init__(message)
+    """
+    This exception is raised if DAC analysis determines that to achieve the 
+    nominal bias current/voltages in ``nominalDacValues`` dictionary, 
+    of: ``gempython.gemplotting.utils.anaInfo``, the DAC would need to be set
+    to a value outside of the DAC range ``[0, max]`` where ``max`` is
+    ``maxVfat3DACSize[dacSelect][0]``.  Info can be found on these ranges in
+    the ``maxVfat3DACSize`` dictionary of: ``gempython.tools.hw_constants``
+    """
```

Comment:
The documentation should describe the current behavior of the exception class.

Reply:
documentation updated